build/tools/SharedCodeGenerator/Program.cs

```diff
@@ -107,8 +107,11 @@ private static void GenerateSupportedPlatformsReadmeFile(string repoDir)
             {
                 sw.WriteLine("# Supported platforms and versions");
                 sw.WriteLine();
-                foreach (var subDirPath in Directory.GetDirectories(platformsDir))
+                string[] subDirs = Directory.GetDirectories(platformsDir);
+                string[] sortedSubDirs = subDirs.OrderBy(n => n).ToArray();
```

Comment:
Nit: feel free to ignore. Could you please clarify why we needed the `.ToArray()`? I would have expected that OrderBy would return an IEnumerable or a similar type, that we could directly use in the foreach.

Reply:
that makes sense! Those were for debugging if the `orderby` did anything. Just changed it with the new commit.